test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/ModelBindingHelperTest.cs

```diff
@@ -46,7 +47,7 @@ public static TheoryData<ModelBindingResult> UnsuccessfulModelBindingData
             var result = await ModelBindingHelper.TryUpdateModelAsync(
                 model,
                 string.Empty,
-                Mock.Of<HttpContext>(),
+                new ActionContext() { HttpContext = new DefaultHttpContext() },
```

Comment:
With this change in, will we have any remaining `Mock<HttpContext>`s? If yes, why?